src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java

```diff
@@ -222,4 +224,21 @@ public void SignUp_008_signupJapaneseUser() {
     preferences.selectTab(tabNames.EMAIL);
     preferences.verifyEmailMeSection();
   }
+
+  @BeforeMethod(alwaysRun = true)
+  public void setEnvironmentCredentials() {
```

Comment:
ok, I really don't think we need to do that since the case of setting credentials for facebook is so specific.  Instead what I did was removed the facebook signup test from signup tests and added it to facebook tests file so that all facebook tests are contained in one area, and this code will only be in one place.